### PR TITLE
HIVE-26943 : Fix NPE during Optimised Bootstrap when db is dropped

### PR DESCRIPTION
**Pull request by shreenidhiSaigaonkar**
### What changes were proposed in this pull request?

This change helps to gracefully fail with proper error message when target db is dropped.

### Why are the changes needed?

Avoids NPE.

### Does this PR introduce _any_ user-facing change?

NO

### How was this patch tested?

* Tested the change in local environment and also in on-prem cluster.

### CODE REVIEW

**Comment by shreenidhiSaigaonkar** [comment]
@pudidic Need review on this too

**Review by ayushtkn — CHANGES_REQUESTED** [review]
Makes sense but requires a test

File: ql/src/java/org/apache/hadoop/hive/ql/exec/repl/ReplLoadTask.java [hunk 1]
```diff
@@ -721,6 +721,10 @@ private int executeIncrementalLoad(long loadStartTime) throws Exception {
     Database targetDb = getHive().getDatabase(work.dbNameToLoadIn);
     Map<String, String> props = new HashMap<>();
 
+    if(targetDb == null) {
```
Comment (pudidic) [inline ql/src/java/org/apache/hadoop/hive/ql/exec/repl/ReplLoadTask.java:724]: Please follow the coding convention; have a whitespace after if.

Comment (shreenidhiSaigaonkar) [reply ql/src/java/org/apache/hadoop/hive/ql/exec/repl/ReplLoadTask.java]: Done.

**Comment by pudidic** [comment]
LGTM. I will merge it.

**Pull request merged**